docker/image_types.yaml

```diff
@@ -0,0 +1,67 @@
+# Defines the different types of docker images that FuzzBench uses and their
+# dependency graph.
+
+'base-image':
+  tag: 'base-image'
+  context: 'docker/base-image'
+
+'base-builder':
+  tag: 'base-builder'
+  context: 'docker/base-builder'
+  depends_on:
+    - 'base-image'
```

Comment:
It also needs `pull-base-clang` from `gcr.io/oss-fuzz-base` according to current `docker/build.mk`. Does BuiltKit will remove this dependency?

Reply:
Yes